docs/docs/workflows/upgrade.md

```diff
@@ -1,44 +1,50 @@
 # Upgrade your cluster
 
-Constellation provides an easy way to upgrade to the next release.
-This involves updating the CLI, choosing a new VM image to use for all nodes in the cluster, and updating the cluster's expected measurements.
+Constellation provides an easy way to upgrade all components of your cluster, without disrupting it's availability.
+Specifically you can upgrade the Kubernetes version, the nodes' image and the Constellation microservices.
```

Comment:
```suggestion
Constellation provides an easy way to upgrade all components of your cluster, without disrupting its availability.
Specifically, you can upgrade the Kubernetes version, the nodes' image, and the Constellation microservices.
```
MS style guide recommends a comma following introductory phrases and also the Oxford comma